### DISCUSSION TITLE
RC522 repeat sensor message

### DISCUSSION THREAD

**Discussion by DarthWeber** [arendst/Tasmota#11639] [Q&A] [answered]
Hi ! I built my tasmota with the rc522 rfid reader, which was pretty simple and runs straight away. Now i get a sensor message with the uid of the card each time i put a card on the reader. 

But is there any way to get those messages periodically as long as the card is hold on the reader or - even better - to get a message, when the card is REMOVED ?

that would increase the usability as i am going to use the rc522 for a musicbox (play a book or playlist as long as the card is laying on the rc522

**Comment by ascillato2** [comment, marked as answer]
Please, address this to the [Tasmota Support Chat](https://discord.gg/Ks2Kzd4). The chat is a better and more dynamic channel for helping you.

Thanks.

---------------------------

### Support Information 
See [Docs](https://tasmota.github.io/docs/) for more information.
See [Chat](https://discord.gg/Ks2Kzd4) for more user experience.
See [Code of Conduct](https://github.com/arendst/Tasmota/blob/development/CODE_OF_CONDUCT.md)

**Comment by mnbrgr** [comment]
I have the exact same question and I wish I would have found the answer here.

**Reply by sfromis** [reply]
Well, life is hard :imp: 
No one posted an answer here, therefore a suggestion to try the Discord server was posted.
